### ISSUE TITLE
Variable Refresh Rate toggle not appearing in Windows Settings

### ISSUE THREAD

**Issue by rx570gtx1060** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#180]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

Windows Settings ( Windows 10)
Then System and then Display.
Then under Graphics Settings

### Processor / Processor Number [Required]

Intel Core i5-10400

### Graphic Card [Required]

Intel Arc A770 16GB Limited Edition GPU

### GPU Driver Version [Required]

- [X] 31.0.101.3793 (*latest*)
- [ ] 31.0.101.3491
- [ ] 31.0.101.3490
- [ ] 31.0.101.3430
- [ ] 31.0.101.3276
- [ ] 30.0.101.3268
- [ ] 30.0.101.3259
- [ ] 30.0.101.1743

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [X] Not applicable

### Windows Build Number [Required]
- [X] Windows 10 22H2
- [ ] Windows 11 22H2
- [ ] Windows 11 21H1 (*October 2021 Update*)
- [ ] Windows 10 21H2 (*November 2021 Update*)
- [ ] Windows 10 21H1 (*May 2021 Update*)
- [ ] Windows 10 20H2 (*October 2020 Update*)
- [ ] Other (Please specify) 

### Other Windows build number

Microsoft Windows 10 22H2 (OS Build 19045.2193)

### Intel System Support Utility report

[bugreportforintelarcissues4.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/9974106/bugreportforintelarcissues4.txt)

### Description and steps to reproduce [Required]

Variable Refresh Rate (VRR) toggle has not been available  to enable under the Latest Driver 31.0.101.3793 (Beta) and the previous driver 31.0.101.3491.
My monitor has FreeSync which is enabled via the Monitor OSD menu.
My monitor is connected to the GPU via a HDMI to HDMI cable.
No HDMI to Displayport adapters/cables have been used.
Originally, I had the 31.0.101.3491 driver which was didn't show any Variable Refresh Rate toggle in the Windows Settings.
So I ran DDU to remove the old driver and try to install the Beta Driver to see if it would fix the problem.
But it hasn't.

System Specifications:
CPU: Intel Core i5-10400
Motherboard: MSI MAG B460 TOMAHAWK (MS-7C81) (Yes Resizable BAR is enabled as well as Above 4G Decoding)
GPU: Intel Arc A770 16GB Limited Edition
RAM: 16GB
Windows Build: 22H2
Monitor: LG 29WK600-W

![Screenshot 2022-11-09 190103](https://user-images.githubusercontent.com/70267266/200918087-c14807b7-ac39-4b23-a426-9785a00c9e9b.png)

![intelgraphicscommandcentre](https://user-images.githubusercontent.com/70267266/200918431-9cb02bd1-33c7-4335-8d8b-f297cf88d87c.png)

![Screenshot 2022-11-09 190440](https://user-images.githubusercontent.com/70267266/200918623-a10dc42e-a4de-462e-a4b3-9f285c3ffcf5.png)

**Comment by AntVNS** [comment]
Hi, 
I have this issue, frist time when install Latest Driver 31.0.101.3793 (Beta) the settings appear. Just now i started my computer see your poste and check my intel graphic command center software toggle has not been available. :(
I reboot my computer and it appear again. Kinda weird...
I do not touch anything, my freesync is on in the monitor OSD , cable is displayport to displayport.

**Comment by kpsam2000Intel** [comment]
Good to check whether you get VRR option when connected using DP port as it's not clear whether this monitor (LG 29WK600-W) supports HDMI 2.1 (standard) based VRR. Monitor spec doesn't call out HDMI2.1 support

**Comment by rx570gtx1060** [comment]
> Hi, I have this issue, frist time when install Latest Driver 31.0.101.3793 (Beta) the settings appear. Just now i started my computer see your poste and check my intel graphic command center software toggle has not been available. :( I reboot my computer and it appear again. Kinda weird... I do not touch anything, my freesync is on in the monitor OSD , cable is displayport to displayport.
> 
> Can you try to install also the Intel® Graphics Command Center (Beta) software in the microsoft store. Open the beta software, normaly you don't see the toggle. Close de soft, and go to windows app and uninstall the beta software. Reboot you computer and check the settings in you Intel Graphics command center.

It still doesn't work, even after trying these steps.

**Comment by rx570gtx1060** [comment]
> Good to check whether you get VRR option when connected using DP port as it's not clear whether this monitor (LG 29WK600-W) supports HDMI 2.1 (standard) based VRR. Monitor spec doesn't call out HDMI2.1 support

Regarding that the HDMI Version is 2.0
Displayport Version is 1.4
I've ordered a Displayport to Displayport cable.

**Comment by IntelSupport-Rozilah** [comment]
hi @rx570gtx1060 ,
Thanks for contacting Intel, we will start to validate the issue once the setup is ready and will update you soon. Stay tuned!

**Comment by rx570gtx1060** [comment]
> hi @rx570gtx1060 , Thanks for contacting Intel, we will start to validate the issue once the setup is ready and will update you soon. Stay tuned!

Thanks.

**Comment by rx570gtx1060** [comment]
> hi @rx570gtx1060 , Thanks for contacting Intel, we will start to validate the issue once the setup is ready and will update you soon. Stay tuned!

I tested the Displayport to Displayport cable I purchased allowed Variable Refresh Toggle to be seen in Windows Settings while FreeSync was enabled on my monitor.

Thanks for your help.

I think it will be a good idea to put that a displayport to displayport cable is needed for adaptive sync on Intel arc support pages.
If it is already there, ignore my comment.
